.github/workflows/cd.yml

```diff
@@ -0,0 +1,26 @@
+name: CD
+
+on:
+  workflow_run:
+    workflows: [CI]
+    types: [completed]
+    branches: [main]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - uses: actions/checkout@8230315d06ad95c617244d2f265d237a1682d445
+      - name: Tag and Push Gem
+        id: tag-and-push-gem
+        uses: discourse/publish-rubygems-action@04549cca4eecd343acd215114ebbbdb99630af90
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          GIT_EMAIL: ${{secrets.ALEX_GIT_EMAIL}}
```

Comment:
Is there a better one for this?

Reply:
I think we could probably set it to something like `dev@gusto.com` or whatever we use nowadays.

If you're okay with shipping this now, I can create a rubyatscale org-wide issue to capture the change to change all references to my email to use the gusto email (perhaps still as a secret just for spam prevention).

FYI this is how its used: https://github.com/search?q=repo%3Adiscourse%2Fpublish-rubygems-action%20GIT_EMAIL&type=code